Mathlib/Data/Set/Lattice.lean

```diff
@@ -543,6 +543,14 @@ theorem directed_on_iUnion {r} {f : ι → Set α} (hd : Directed (· ⊆ ·) f)
     ⟨x, ⟨z, xf⟩, xa₁, xa₂⟩
 #align set.directed_on_Union Set.directed_on_iUnion
 
+theorem directed_on_sUnion {r} {S : Set (Set α)} (hd : DirectedOn (· ⊆ ·) S)
```

Comment:
This should be `directedOn__sUnion`. The one above is wrong too - if it's not used too much, you might rename it in this PR, leaving a deprecated alias. Alternatively leave a TODO comment

Reply:
Done (and moved to line 2037), see comment below.